s3storage.py

```diff
@@ -126,7 +126,7 @@ def retrieve_object_body(self, key):
                 Bucket=self.bucket, Key=key)
             content = res.get('Body').read().decode('utf-8')
         except botocore.exceptions.ClientError as err:
-            logging.error(err)
+            logging.debug(err)
```

Comment:
these errors are basically swallowed, (invisible if the debug logs are not printed) 🤔 is it part of a try loop?

Reply:
Thanks! Fixed